lib/liquid_voting/release.ex

```diff
@@ -11,6 +13,23 @@ defmodule LiquidVoting.Release do
     {:ok, _, _} = Ecto.Migrator.with_repo(repo, &Ecto.Migrator.run(&1, :down, to: version))
   end
 
+  @doc """
+  Deletes votes, participants and delegations data created by smoke tests.
+  For use before and after running smoke tests.
+  """
+  def teardown() do
+    organization_id = "bc7eeccb-5e10-4004-8bfb-7fc68536bbd7"
+
+    votes = Voting.list_votes(organization_id)
+    Enum.each(votes, fn vote -> Voting.delete_vote!(vote) end)
+
+    participants = Voting.list_participants(organization_id)
+    Enum.each(participants, fn participant -> Voting.delete_participant(participant) end)
```

Comment:
We should probably use the `!` version of this (or pattern match on the response at the very least)?